lua/nvim-treesitter/install.lua

```diff
@@ -66,6 +70,14 @@ local function run_install(cache_folder, package_path, lang, repo, with_sync)
   -- compile_location only needed for typescript installs.
   local compile_location = cache_folder..'/'..(repo.location or project_name)
   local parser_lib_name = package_path.."/parser/"..lang..".so"
+
+  local compilers = { "cc", "gcc", "clang" }
```

Comment:
Should we export the compilers of this module, so that users can set their favorite compiler in a weird location?

Reply:
Hmmm.. Maybe at some point, but let's keep it like that for now